### PR TITLE
added country where the people live (according to their linked-in profile)

### PR DESCRIPTION
**Pull request by AndiSwiss**
…files)

### CODE REVIEW

File: src/views/Team.vue [hunk 1]
```diff
@@ -93,73 +94,84 @@ export default {
...
           linkedin: 'https://www.linkedin.com/in/andreas-ambuehl',
           github: 'https://github.com/AndiSwiss',
+          country: 'Switzerland',
           picture: '/pics/profile_andreas_ambuehl.jpg',
         },
         {
           name: 'Apostolos Staridis',
           role: 'QA & Marketing',
           linkedin: 'https://www.linkedin.com/in/aposstar1994',
+          country: 'Greece',
```
Comment (PaschalisDim) [inline src/views/Team.vue]: UK here

Comment (AndiSwiss) [reply src/views/Team.vue]: ok, I'll change it (I just pulled the info from linkedin)

**Comment by AndiSwiss** [comment]
> Does hanna live in GB?

I'll fix that (was a copy-paste-error)

File: src/views/Team.vue (hunk 1 above)
Comment (AndiSwiss) [reply src/views/Team.vue]: I implemented all the changes and pushed again... should shortly be ready to merge

**Comment by AndiSwiss** [comment]
> Does hanna live in GB?

Fixed in newest push

**Pull request merged**